R/discrete_table.R

```diff
@@ -96,8 +96,8 @@ cat_data <- function(data, cols, by = ".total", panel = by,
       ans <- complete(ans, !!!syms(.groups))
       nstart <- length(.groups)
       nend <- ncol(ans)
-      ans <- mutate(ans, across(nstart+1, replace_na, 0))
-      ans <- mutate(ans, across(seq(nstart+2, nend), replace_na, "0 (0.0)"))
+      ans <- mutate(ans, across(nstart+1, ~ replace_na(.x, 0)))
+      ans <- mutate(ans, across(seq(nstart+2, nend), ~ replace_na(.x, "0 (0.0)")))
```

Comment:
The change looks fine, and I see that the `~` purrr shorthand is used elsewhere is this package, so this is just my two cents:  in the package context, I dislike that `~`, compared to `function(...) ...` (or `\(x) ...` for R 4.1 or later),  prevents `R CMD check` from "seeing inside" the closure and warning about things (e.g., undefined variable).

Reply:
Thanks. Changed `~` to regular function.